### PR TITLE
dev: add bash utility script for docker

### PR DESCRIPTION
**Pull request by yuri-rage**
Bash utility script to encapsulate the lengthy `docker run` command for ease of use. Presents a brief menu of convenience options to the user if run without arguments.

### CODE REVIEW

File: docker_update_py.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    else
+        fast_option=""
+    fi
+
+    echo -e "\nRunning: ${BOLD}${GREEN}update.py $fast_option $site_option${NC}\n"
+    run_command $fast_option $site_option
+}
+
+run_command() {
+    docker run --rm -it -v "${PWD}:/ardupilot_wiki" -u "$(id -u):$(id -g)" ardupilot_wiki python3 update.py "$@"
```
Comment (Ryanf55) [inline docker_update_py.sh:43]: Can you have it automatically build the image if it doesn't exist?

Comment (Ryanf55) [inline docker_update_py.sh:43]: Can you have it automatically build the image if it doesn't exist?

File: common/source/docs/common-wiki-editing-setup.rst [hunk 2]
```diff
@@ -122,6 +122,15 @@ This will build a docker image with all package setup to build the wiki and name
...
        docker run --rm -it -v "${PWD}:/ardupilot_wiki" -u "$(id -u):$(id -g)" ardupilot_wiki python3 update.py
 
+#. Alternately, use the bash utility script in the ``ardupilot_wiki`` root directory:
+
+   .. code-block:: bash
+
+       ./docker_update_py.sh  # presents a brief menu of build options
+
+       # or, to build the copter site with the --fast flag, for example:
+       ./docker_update_py.sh --fast --site copter
```
Comment (Ryanf55) [inline common/source/docs/common-wiki-editing-setup.rst:132]:
```suggestion
       # presents a brief menu of build options
       ./docker_update_py.sh  

       # or, to build the copter site with the --fast flag, for example:
       ./docker_update_py.sh --fast --site copter
```

This makes copy-paste easier if it's on 2 lines.

**Review by Ryanf55 — CHANGES_REQUESTED** [review]
The dockerfile might be broken. Unrelated to the PR, but I can't test it till the python requirements are installed.

```
$  ./docker_update_py.sh

Select ArduPilot Wiki site to build:

  1) Copter
  2) Plane
  3) Rover
  4) Dev
  5) All

Enter choice [1-5]: 4

Build fast? (Y/n): y

Running: update.py --fast --site dev

Traceback (most recent call last):
  File "/ardupilot_wiki/update.py", line 49, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
```

File: docker_update_py.sh (hunk 1 above)
Comment (Ryanf55) [reply docker_update_py.sh:43]: If you don't have the image, you get this:
```
Running: update.py --fast --site dev

Unable to find image 'ardupilot_wiki:latest' locally
docker: Error response from daemon: pull access denied for ardupilot_wiki, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
```

Comment (yuri-rage) [reply docker_update_py.sh:58]: Added.

Comment (Ryanf55) [reply docker_update_py.sh:58]: I think it's ok to duplicate. We don't need  to force people to use this script (yet).

**Review by Ryanf55 — APPROVED** [review]
Tested without docker image, building a few different sites fast and slow. Works great! No more forgetting to source my virtual environment or opening the wiki to get the docker commands to run to build the wiki.

**Pull request merged**